docs/sources/installation/archlinux.md

```diff
@@ -59,3 +59,13 @@ To start on system boot:
 If you need to add an HTTP Proxy, set a different directory or partition for the
 Docker runtime files, or make other customizations, read our systemd article to
 learn how to [customize your systemd Docker daemon options](/articles/systemd/).
+
+## Uninstall Docker
+
+To uninstall the Docker package:
+
+	pacman -R docker
+
+To uninstall the Docker package and dependencies that are no longer needed:
+
+	pacman -Rns docker
```

Comment:
Same here.

Also (not sure) indentation looks to be > 4 spaces (here and line 67)